### PR TITLE
Adds three tables to the package list

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-2964](https://openedx.atlassian.net/browse/DOC-2964)

The list of files included in the compressed data package got stale over time. This PR updates the list in package.rst. Then, I got carried away and cleaned up some inconsistent date formatting in the sql_schema.rst file.
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @stroilova 
- [x] Doc team review (sanity check): @srpearce or @pdesjardins  
### Testing
- [x] Ran ./run_tests.sh without warnings or errors
### Post-review
- [x] Squash commits

### CODE REVIEW

File: en_us/data/source/data_czars/package.rst [hunk 1]
```diff
@@ -225,27 +226,48 @@ on the site. For more information, see :ref:`course_structure`.
...
 course. See :ref:`student_courseenrollment`.
 
+``{org}-student_languageproficiency-{site}-analytics.sql``
+************************************************************************
+
+Indicates each learner's self-reported language preference. See :ref:`student_languageproficiency`.
+
+Learners have the option to indicate a preferred language on the dashboard profile page. For more information, see :ref:`partnercoursestaff:SFD Profile Page`.
+
+.. ^^Olga, I think that this data is collected on the learner dashboard > profile page, so it is not course-specific. Is this the correct name for the file, and is it found in this zip file?? (Alison move this 2nd sentence and xref to sql_schema if correct.)
```
Comment (lamagnifica) [inline en_us/data/source/data_czars/package.rst]: @stroilova could you please check these questions?

File: en_us/data/source/internal_data_formats/sql_schema.rst [hunk 2]
```diff
@@ -1139,6 +1139,8 @@ self-reported language preferences. Learners can select only one value.
 | code            | varchar(16) | NO   | MUL | NULL    |                |
 +-----------------+-------------+------+-----+---------+----------------+
 
+.. ^^Olga, by default the language dropdown on the profile page is not filled and is blank. is there a code for that setting, or is the last line of this table incorrect and the code field can be Null?
```
Comment (stroilova) [inline en_us/data/source/internal_data_formats/sql_schema.rst]: @lamagnifica I'm not seeing either blanks or NULLS in the student_languageproficiency table. 

Perhaps if no language preference is selected, the student just does not get a row in the table?
We'd need to confirm with the Dev team to be sure.

@nasthagiri  (or someone else from TNL) - does that sound right? Thanks.

**Comment by stroilova** [comment]
In general, 👍 

On the language proficiency defaults question, I gave a summary and asked TNL for a confirmation.

File: en_us/data/source/data_czars/package.rst [hunk 3]
```diff
@@ -225,27 +226,52 @@ on the site. For more information, see :ref:`course_structure`.
...
+
+Indicates each learner's self-reported language preference. See
+:ref:`student_languageproficiency`.
+
+Learners have the option to indicate a preferred language on the dashboard
+profile page. For more information, see :ref:`partnercoursestaff:SFD Profile
+Page`.
+
+.. ^^Olga, I think that this data is collected on the learner dashboard > profile page, so it is not course-specific. Is this the correct name for the file, and is it found in this zip file?? (Alison move this 2nd sentence and xref to sql_schema if correct.)
+
```
Comment (lamagnifica) [inline en_us/data/source/data_czars/package.rst]: @stroilova can you confirm the file name format?

File: en_us/data/source/data_czars/package.rst [hunk 4]
```diff
@@ -225,27 +226,52 @@ on the site. For more information, see :ref:`course_structure`.
...
+``{org}-{course}-{run}-teams_courseteam-{site}-analytics.sql``
+************************************************************************
+
+Identifies the teams of learners established in a course that uses the teams
+feature. See
```
Comment (srpearce) [inline en_us/data/source/data_czars/package.rst]: Nit: Line wrap weirdness.

**Comment by srpearce** [comment]
One tiny nit about line wrap weirdness that you don't actually have to fix. 👍

File: en_us/data/source/internal_data_formats/sql_schema.rst (hunk 2 above)
Comment (nasthagiri) [inline en_us/data/source/internal_data_formats/sql_schema.rst]: That's correct.  The table contains rows for only students who have selected a language preference.

You are probably already aware of this, but please note that we have 2 different language settings on the user Account settings page:
1. "Language" is for specifying the user's language for translating all the content on the site.
2. "Preferred Language" is for specifying the user's language on the user's profile.  The platform does not use this value for translation purposes. It is only for sharing with other learners.

File: en_us/data/source/internal_data_formats/sql_schema.rst [hunk 5]
```diff
@@ -1125,9 +1125,11 @@ Columns in the ``student_languageproficiency`` Table
 ====================================================
 
 The ``student_languageproficiency`` table stores information about learners'
-self-reported language preferences. Learners can select only one value.
+self-reported language preferences. Learners have the option to indicate a
+preferred language on the dashboard profile page. Learners can select only one
+value. For more information, see :ref:`partnercoursestaff:SFD Profile Page`.
```
Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/sql_schema.rst:1130]: @nasthagiri wrote: (sorry, squashed commits and rebased at a bad time):

That's correct. The table contains rows for only students who have selected a language preference.

You are probably already aware of this, but please note that we have 2 different language settings on the user Account settings page:
1. "Language" is for specifying the user's language for translating all the content on the site.
2. "Preferred Language" is for specifying the user's language on the user's profile. The platform does not use this value for translation purposes. It is only for sharing with other learners.

Comment (lamagnifica) [inline en_us/data/source/internal_data_formats/sql_schema.rst:1130]: @nasthagiri and @stroilova on stage I see the following when I select the dropdown symbol next to my user name:
1.  If I select Profile, I can "Add language" <-- where is this stored? what is it used for? (this is what I thought got stored in the student_languageproficiency table!)
2.  If I select Account I can select 
   -  Language <-- where is this stored? what is it used for? is this what you mean by your #1, and are you saying that is is stored in the student_languageproficiency table?
   - Preferred Language <-- where is this stored? what is it used for? is this your #2 above, and are you saying that it is _not_ stored in the student_languageproficiency table?

Comment (nasthagiri) [inline en_us/data/source/internal_data_formats/sql_schema.rst:1130]: @lamagnifica 
Tip: You can tell which one is which by seeing how many language options are available.  The menu with fewer options is the menu for the translation Language.
1. The Profile Language is stored in the `student_languageproficiency` table.
2. Yes. "Language" is my #1.  It is stored in the user's Preferences table.  "Preferred Language" is my #2 and is stored in the `student_languageproficiency` table.

**Pull request merged**